extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -311,27 +316,50 @@ export async function promptForBindingType(funcName?: string): Promise<BindingTy
 	return selectedBinding?.type;
 }
 
+/**
+ * Prompts the user to select to use a table or view as the object to query/upsert into
+ */
+export async function promptForObjectType(): Promise<string | undefined> {
+	const inputOutputItems: (vscode.QuickPickItem)[] = [
```

Comment:
You can just pass in an array of strings directly if you don't need to have additional information, no need to use QuickPickItems for that.